lib/dbservice/grades.ex

```diff
@@ -59,6 +60,7 @@ defmodule Dbservice.Grades do
   def create_grade(attrs \\ %{}) do
     %Grade{}
     |> Grade.changeset(attrs)
+    |> Ecto.Changeset.put_assoc(:group, [%Group{type: "grade", child_id: attrs["id"]}])
```

Comment:
are u sure these are the only changes? what about group_view.ex ... that file has a rendering for `auth_group` .. shouldn't we have similar code for `grade`? assuming we treat `grade` as group everywhere

Reply:
It's upto us based on our requirement whether we want to return grade details along with group or not. Added it for now in case we might require it somewhere.